### PR TITLE
Implement Home Button and Sustain Title

### PR DESCRIPTION
**Pull request by zaidokal**
# JIRA Ticket Link

https://skaarkzo-capstone.atlassian.net/browse/SUSTAIN-43

# Summary of Changes

Added Home Button to redirect to home page
Added Sustain title

### CODE REVIEW

**Review by arhamansarii — APPROVED** [review]

File: sustain/src/app/page.tsx [hunk 1]
```diff
@@ -1,101 +1,17 @@
...
-          />
-          Go to nextjs.org →
-        </a>
-      </footer>
+    <div
+      className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20"
+      style={{ backgroundColor: "rgb(37, 37, 37)" }}
+    >
+      <div className="flex flex-col items-center justify-center min-h-screen text-white">
+        <h3 className="text-[21px] mb-4">
```
[the remark this answers is not in the document]
Comment (zaidokal) [reply sustain/src/app/page.tsx:10]: Added the spaces between letters

File: sustain/src/app/page.tsx [hunk 2]
```diff
@@ -1,101 +1,17 @@
...
+          <Link href="/">Home</Link>
+        </h3>
+        <h1 className="text-[67px] font-aboreto">SUSTAIN</h1>
```
[the remark this answers is not in the document]
Comment (zaidokal) [reply sustain/src/app/page.tsx]: Added the green color to AI part as agreed in chat

File: sustain/src/app/page.tsx [hunk 3]
```diff
@@ -1,101 +1,19 @@
...
+    <div
+      className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20"
+      style={{ backgroundColor: "rgb(37, 37, 37)" }}
+    >
+      <div className="flex flex-col items-center justify-center min-h-screen text-white">
+        <h3 className="text-[21px] mb-4">
+          <Link href="/">Home</Link>
+        </h3>
+        <h1 className="text-[67px] font-aboreto">
+          S U S T <span style={{ color: "rgb(30, 106, 39)" }}>A I </span>N
```
Comment (rakhshankhan) [inline sustain/src/app/page.tsx:14]: There's gotta be a better way to add the spacing that isn't hard coded. Try checking if there's a tailwind attribute cuz default CSS has one.

Comment (arhamansarii) [reply sustain/src/app/page.tsx:14]: yes it has attributes: https://tailwindcss.com/docs/letter-spacing

Comment (arhamansarii) [inline sustain/src/app/page.tsx:14]: The font should be thinner. Also doesn't seem to match with the style on Figma?

Comment (zaidokal) [reply sustain/src/app/page.tsx]: Yup, fixed the font in the new commit

Comment (zaidokal) [reply sustain/src/app/page.tsx]: Done, used a custom spacing as seen in the link. Thanks for the resource!

**Review by arhamansarii — APPROVED** [review]

**Review by rakhshankhan — APPROVED** [review]

**Pull request merged**